client/components/thank-you/index.tsx

```diff
@@ -151,26 +147,20 @@ export const ThankYou = ( props: ThankYouProps ): JSX.Element => {
 			<ThankYouHeader className={ classNames( 'thank-you__container-header', headerClassName ) }>
 				{ /* eslint-disable-next-line jsx-a11y/alt-text */ }
 				<img { ...thankYouImage } />
-				<ThankYouFooterContainer>
-					<h1> { thankYouImageFooter } </h1>
-					<h2> { thankYouImageFooterSubtitle } </h2>
-				</ThankYouFooterContainer>
+				{ thankYouTitle && (
+					<ThankYouTitleContainer>
+						<h1 className="thank-you wp-brand-font"> { thankYouTitle } </h1>
```

Comment:
I think we should use a more specific class name for this. I also think we can remove the whitespace outside of the `{}` wrapper.
```suggestion
						<h1 className="thank-you__container_header wp-brand-font">{ thankYouTitle }</h1>
```